index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <header id="header">
+      <div>
+        <img id="header-img" class="mini img" src="https://media.istockphoto.com/vectors/continuous-one-line-drawing-of-trumpet-music-instrument-simple-vector-id1316835411?k=20&m=1316835411&s=612x612&w=0&h=wt7nT-cDY9AjfmXivld3FT9iX-M8du6-VFi94aHKTak=" alt="Company Logo">
+      </div>
+      <div>  
+      <nav id="nav-bar">
+        <ul>
+          <li><a class="nav-link" href="#feat">Features</a> </li>
+          <li><a class="nav-link" href="#video">How It Works</a> </li>
+          <li><a class="nav-link" href="#price">Pricing</a></li>
+        </ul>
+      </nav>
+      </div>
+    </header>
+   
+   <form id="form" action="https://www.freecodecamp.com/email-submit" method="attribute">
+     <input id="email" name="email" type="email" placeholder="Enter your email" required>
+     <input id="submit" type="submit" value="Let's begin" class="bts"/>
+   </form>
+   <br>

```

Comment:
- [ ] I think it would be better if you position the `form` element using CSS code instead using the `br` tags and deleting it. It helps to get a cleaner code. 👽